archive-path.py

```diff
@@ -61,6 +61,6 @@ def setting(key):
             raise Exception(
                 u"Warning: Cannot get The Archive setting: {0}".format(key))
 
-
```

Comment:
These two lines were only added to demonstrate how to use. You can just remove them!